### PR TITLE
CHE-514: fix jdt tests

### PR DESCRIPTION
**Pull request by svor**

### CODE REVIEW

**Comment by vparfonov** [comment]
ok

File: plugins/plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/resources/projects/test/src/main/java/com/codenvy/test/MyClass.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
 
-package org.eclipse.che.test;
+package com.codenvy.test;
```
Comment (skabashnyuk) [inline plugins/plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/resources/projects/test/src/main/java/com/codenvy/test/MyClass.java:13]: why?

Comment (svor) [inline plugins/plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/test/resources/projects/test/src/main/java/com/codenvy/test/MyClass.java:13]: I've already fixed it.

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**